### PR TITLE
chore: upgrade polkadot packages

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Related #4585

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at ec7e454</samp>

This pull request improves the node modules compatibility, the browser targeting, and the Polkadot dependencies of the project. It modifies the `nuxt.config.js`, `.browserslistrc`, and `package.json` files accordingly.

### <samp>🤖 Generated by Copilot at ec7e454</samp>

> _We are the NFT warriors, we defy the old and stale_
> _We update our dependencies, we optimize our code_
> _We use the Polkadot API, we display our identities_
> _We tweak our webpack config, we rule the smoldot-light_

### CODE REVIEW

**Comment by preschian** [comment]
the issue still exists, I will check in the other packages. changes in this PR:
- fix loader for a newer `@polkadot/*` packages
- unify `.mjs` loader
- upgrade `@paraspell/sdk` to a newer version. how to test this btw?

Screenshot 2023-05-05 154604

**Comment by roiLeo** [comment]
>     * upgrade `@paraspell/sdk` to a newer version. how to test this btw?

This is used in `/teleport-bridge` page

**Comment by preschian** [comment]
> > ```
> > * upgrade `@paraspell/sdk` to a newer version. how to test this btw?
> > ```
> 
> This is used in `/teleport-bridge` page

seems stuck on the rpc is pending on my end. I check it on canary

**Comment by vikiival** [comment]
the issue still exists, I will check in the other packages. changes in this PR:

> fix loader for a newer @polkadot/* packages

please also update packages in sub-api (kodadot/packages)

> unify .mjs loader

Won't this cause slower build times?

> upgrade @paraspell/sdk to a newer version. how to test this btw?

- https://github.com/paraspell/xcm-sdk/issues/34

**Comment by vikiival** [comment]
Pushed new @paraspell sdk 

also opened: 
- paraspell/sdk#35

**Comment by preschian** [comment]
> please also update packages in sub-api (kodadot/packages)

related PR https://github.com/kodadot/packages/pull/137

> Won't this cause slower build times?

seems not. but, let me know if you found out that it makes the build slower

File: package.json [hunk 1]
```diff
@@ -76,12 +76,16 @@
     "@nuxtjs/i18n": "^7.3.1",
     "@nuxtjs/sentry": "^7.2.2",
     "@oruga-ui/oruga": "^0.5.10",
-    "@paraspell/sdk": "0.0.25",
+    "@paraspell/sdk": "^1.1.4",
     "@pinia/nuxt": "^0.4.9",
+    "@polkadot/api": "^10.6.1",
+    "@polkadot/api-base": "^10.6.1",
+    "@polkadot/apps-config": "^0.129.1",
```
Comment (roiLeo) [inline package.json:83]: new deps? are they needed?

**Review by roiLeo — APPROVED** [review]
looks oki, we might have to check if transpile is still needed

File: package.json (hunk 1 above)
Comment (preschian) [reply package.json:83]: yes, based on their instruction https://github.com/paraspell/xcm-sdk#install-deps-via-npmyarnpnpm

if I removed those packages will error

```
ERROR in ./node_modules/.pnpm/@paraspell+sdk@1.1.4/node_modules/@paraspell/sdk/dist/index.mjs
Module not found: Error: Can't resolve '@polkadot/apps-config/endpoints' in
```

**Comment by vikiival** [comment]
Fresh from KodaDot bakery:
cc @preschian 

- @kodadot1/minimark@0.1.9
- @kodadot1/sub-api@0.1.2

**Comment by vikiival** [comment]
vuex-options were already released to main because of the breaking change :)

**Review by vikiival — APPROVED** [review]

**Pull request merged**